src/main/java/edu/whimc/observations/models/Observation.java

```diff
@@ -67,10 +68,19 @@ protected Observation(Observations plugin, int id, Timestamp timestamp, String p
         });
     }
 
-    public static Observation createObservation(Observations plugin, Player player, Location viewLoc,
-                                                String observation, Timestamp expiration, ObservationTemplate template) {
+    public static Observation createObservationEventWithCurrentTime(Observations plugin, String observation, Player player, ObservationTemplate template) {
```

Comment:
I was wrong, I think it would be better to just have this method be named `createPlayerObservation` with a JavaDoc explaining that it uses the configured expiration and calls the event; the name is pretty long.

Also, this is a nitpick but can you refactor the order of parameters and swap `observation` and `player` to be consistent with `loadObservation`